.travis.yml

```diff
@@ -4,8 +4,17 @@ env:
   - TEST_SUITE=test_core
   - TEST_SUITE=test_site
 script: "ant $TEST_SUITE"
+addons:
+  apt:
+    packages:
+      - openjdk-6-jdk
+      - ant
+      - ant-optional
 jdk:
-  - openjdk7
+  # prod, test: Java HostSpot 64-bit Server VM;
+  #     Java(TM) SE Runtime Environment (build 1.6.0_33-b03)
+  - openjdk6
+#  - openjdk7  # this was being tested, but now causes failures
```

Comment:
you figured this out before but it never made it to a PR?  huh.  Glad to see it, in any case.